indy-besu/docs/roadmap.md

```diff
@@ -0,0 +1,42 @@
+# Hyperledger Indy Besu ROADMAP
+
+Note: Right now we have finished PoC implementation. Roadmap tasks and their priorities are still in process of defining.
+
+## Phase 0: PoC
```

Comment:
I think it's not clear what DID method we support in PoC, and how it will be evolved in MVP.

Reply:
Added Indy2 DID method mention